src/MarginTrading.Backend.Core/Caches/IOvernightSwapCache.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace MarginTrading.Backend.Core
+{
+	public interface IOvernightSwapCache
+	{
+		OvernightSwapCalculation Get(string key);
+		bool TryGet(string key, out OvernightSwapCalculation item);
+		IReadOnlyList<OvernightSwapCalculation> GetAll();
+		bool TryAdd(OvernightSwapCalculation item);
+		bool AddOrReplace(OvernightSwapCalculation item);
+		void SetAll(IEnumerable<OvernightSwapCalculation> items);
+		void Clear(string key);
+		void ClearAll();
+		void Initialize(IEnumerable<OvernightSwapCalculation> items);
```

Comment:
Difference between Initialize and SetAll?

Reply:
No difference in this specific case. Removed unused methods from interface.